clang/lib/CIR/CodeGen/CIRGenOpenACCClause.h

```diff
@@ -82,6 +83,17 @@ class OpenACCClauseCIREmitter final
     return conversionOp.getResult(0);
   }
 
+  mlir::Value createConstantInt(mlir::Location loc, unsigned width,
+                                int64_t value) {
+    mlir::IntegerType ty = mlir::IntegerType::get(
+        &cgf.getMLIRContext(), width,
+        mlir::IntegerType::SignednessSemantics::Signless);
+    auto constOp = builder.create<mlir::arith::ConstantOp>(
+        loc, ty, builder.getIntegerAttr(ty, value));
```

Comment:
nit: no need for type here:
```suggestion
        loc, builder.getIntegerAttr(ty, value));
```

Reply:
Ah, thanks!